lib/netext/grpcext/conn.go

```diff
@@ -26,16 +26,20 @@ import (
 	"google.golang.org/protobuf/reflect/protoreflect"
 	"google.golang.org/protobuf/types/descriptorpb"
 	"google.golang.org/protobuf/types/dynamicpb"
+	"google.golang.org/protobuf/types/known/emptypb"
 )
 
+var emptyDescriptor = (&emptypb.Empty{}).ProtoReflect().Descriptor() //nolint:gochecknoglobals
```

Comment:
Having this global might lead to a data race. I don't know how `Descriptor` is used by gRPC internally. But if they write some field then when k6 runs with multiple VUs it will generate a data race.

And, even if they are not doing it now, it might happen in the future if they apply changes. I don't think the API makes the guarantee to be concurrent-safe.

So we need to have a dedicated copy for it. Make sure to re-run the benchmark to see if the changes are not dependent on this specific global approach.

Reply:
Yes, intuitively it looked safe, but I agree with the comment. Benchmarks didn't show much of a difference.